### DISCUSSION TITLE
E77 –  Найцікавіші новини місяця: лютий 2024

### DISCUSSION THREAD

**Discussion by malor** [shopokodu/community#52] [Теми для подкасту]
Посилання на новини:

- [x] [Go 1.22 is released!](https://go.dev/blog/go1.22)
- [x] Розчарування від [Sway 1.9](https://github.com/swaywm/sway/releases/tag/1.9)
- [x] Семантичний промпт та SIXEL в [Tmux 3.4](https://raw.githubusercontent.com/tmux/tmux/3.4/CHANGES)
- [x] Білий Дім [закликає не писати на C та C++](https://www.whitehouse.gov/oncd/briefing-room/2024/02/26/press-release-technical-report/)
- [ ] [HDMI проти відкритого коду](https://www.phoronix.com/news/HDMI-2.1-OSS-Rejected)
- [x] [Google дав милостиню Rust](https://foundation.rust-lang.org/news/google-contributes-1m-to-rust-foundation-to-support-c-rust-interop-initiative/)
- [x] [Pingora](https://blog.cloudflare.com/pingora-open-source) від Cloudflare
- [ ] [Introducing Sudo for Windows!](https://devblogs.microsoft.com/commandline/introducing-sudo-for-windows/)
- [ ] [RSS is still pretty great](https://www.pcloadletter.dev/blog/rss/)
- [ ] [Елмо подає в суд на CEO OpenAI](https://www.reuters.com/legal/elon-musk-sues-openai-ceo-sam-altman-breach-contract-2024-03-01/)

**Comment by CakesTwix** [comment]
[KDE 6](https://kde.org/announcements/megarelease/6/), [LineageOS 21](https://lineageos.org/Changelog-28/)
